### PR TITLE
GPVTG Message and Unit Test

### PR DESCRIPTION
**Pull request by fchapleau**

### CODE REVIEW

**Comment by terribletim** [comment]
@fchapleau Thanks for GPVTG message. I note that while you have allowed for optional 8th token (FixType) introduced at NMEA 2.3, this is not being decoded.

File: src/NmeaParser.Shared/Nmea/Gps/GPVTG.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        /// <summary>
+        /// Speed in Knots
+        /// </summary>
+        public double? SpeedInKnots { get; private set; }
+
+        /// <summary>
+        /// Speed in KM per Hour
+        /// </summary>
+        public double? SpeedInKmPerHour { get; private set; }
```
Comment (dotMorten) [inline src/NmeaParser.Shared/Nmea/Gps/GPVTG.cs]: Why are all the variable nullable? It doesn't seem to be possible to get null, and so far the pattern in the library is that doubles are NaN when not available/invalid.

File: src/NmeaParser.Tests/NmeaMessages.cs [hunk 2]
```diff
@@ -374,5 +374,18 @@ public void TestGpgst()
 			Assert.AreEqual(0.020, gst.SigmaLongitudeError);
 			Assert.AreEqual(0.031, gst.SigmaHeightError);
 		}
-	}
+
+        [TestMethod]
+        public void TestGpvtg()
+        {
+            string input = "$GPVTG,331.21,T,,M,0.28,N,0.51,K,A*31";
```
Comment (dotMorten) [inline src/NmeaParser.Tests/NmeaMessages.cs]: 👍

File: src/NmeaParser.Shared/Nmea/Gps/GPVTG.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+        /// <summary>
+        /// Track made good, relative to true North
+        /// </summary>
+        public double? TrueTrackMadeGood { get; private set; }
+
+        /// <summary>
+        /// Track made good, relative to Magetic North
+        /// </summary>
+        public double? MageticTrackMadeGood { get; private set; }
```
Comment (dotMorten) [inline src/NmeaParser.Shared/Nmea/Gps/GPVTG.cs]: Typo in name

**Comment by dotMorten** [comment]
Just realized GPVTG is already there. D'oh! Closing.

**Pull request closed without merging**